tests/smoke/desiredstate_test.go

```diff
@@ -66,8 +66,11 @@ func TestDesiredState(t *testing.T) {
 	require.NoError(t, err)
 	require.Contains(t, stderr, "The installation is out-of-sync, running the install action")
 	require.Contains(t, stderr, "Triggering because the installation has not completed successfully yet")
-	test.RequireInstallationExists("operator", "mybuns")
+	installation := test.RequireInstallationExists("operator", "mybuns")
+	require.Equal(t, "succeeded", installation.Status.ResultStatus)
 
+	_, err = test.ShowInstallation("operator", "mybuns")
```

Comment:
It's not clear what you were testing when you added the call to show installation here?

Reply:
Should be fixed now!